dist/src/core/delimiters.js

```diff
@@ -42,7 +42,7 @@ const makeVlist = Span.makeVlist;
  * @private
  */
 function makeSmallDelim(type, delim, style, center, context, classes) {
-    const text = makeSymbol('AMS-Regular', Definitions.getValue('math', delim));
+    const text = makeSymbol('Main-Regular', Definitions.getValue('math', delim));
```

Comment:
This was using the wrong font to get the size metric for the √ character, so it was falling back to default values, which were inaccurate, especially when the sqrt was inside of a fraction

Reply:
this was the main cause for the sqrt symbol 's top not aligning when in a fraction

![image](https://github.com/purdue-tlt/mathlive/assets/689209/335a017a-7b5e-4f53-bc8b-5ae8935a56ae)

changed to

![image](https://github.com/purdue-tlt/mathlive/assets/689209/fc27dbee-2be2-445c-a354-d4abe4f0d74f)